src/insights/competition.ts

```diff
@@ -3,13 +3,32 @@ import { PageMeta, TimeMeta } from "./common";
 export interface CompetitionItem {
   id: string;
   competitionId: string;
+  subCompetitionId: string;
   market: string;
   start: string; // string representation of timestamp
   end: string; // string representation of timestamp
   enablePNL: boolean;
   minBalance: number;
   enableVolume: boolean;
   makerIncentive: boolean;
+  config: CompetitionItemConfig;
+}
+
+export interface CompetitionItemConfig {
+  type: string;
+  lending: string[];
+  markets: string[];
+  promoMarkets: string[];
+  preTradeMarket: string[];
+  minBalance: number;
+  volPerPoint: number;
```

Comment:
`volPerPoint` is a number string
<img width="302" alt="Screenshot 2024-07-10 at 5 32 31 PM" src="https://github.com/Switcheo/carbon-js-sdk/assets/65583522/86b33868-3e8f-4afd-9fc6-d104a574dbe5">